app/controllers/bike_share_app.rb

```diff
@@ -41,7 +41,7 @@ class BikeShareApp < Sinatra::Base
   end
 
    put '/stations/:id' do
-    @station = Station.update(params[:id], params[:station])
+    @station = Station.record_update(params[:id], params[:station])
```

Comment:
instead of using the built in active record "update" this calls on a class method "record_update" that accepts an argument of "city_name" and saves the "city_id" value into the station model.